lib/Basic/TargetInfo.cpp

```diff
@@ -130,7 +134,18 @@ void targetinfo::printTripleInfo(
   out << "\",\n";
 
   out << "    \"platform\": \"" << getPlatformNameForTriple(triple) << "\",\n";
-  out << "    \"arch\": \"" << swift::getMajorArchitectureName(triple) << "\",\n";
+  out << "    \"arch\": \"" << swift::getMajorArchitectureName(triple)
+      << "\",\n";
+
+  clang::DiagnosticsEngine DE{new clang::DiagnosticIDs(),
+                              new clang::DiagnosticOptions(),
+                              new clang::IgnoringDiagConsumer()};
+  std::shared_ptr<clang::TargetOptions> TO =
+      std::make_shared<clang::TargetOptions>();
+  TO->Triple = triple.str();
+  clang::TargetInfo *TI = clang::TargetInfo::CreateTargetInfo(DE, TO);
+  out << "    \"pointerWidthInBits\": "
```

Comment:
Could we have a test for this in `test/Driver/print_target_info.swift`, since that's where this information is being exposed ?

Reply:
Aye, that is a good idea